unit_test/data_driven_software_tests/test_tests.py

```diff
@@ -0,0 +1,327 @@
+"""Tests of NeuronUnit test classes"""
```

Comment:
@russelljjarvis Have you run these yet?  Do they work?

Reply:
Yes, those where the tests I brought up in the lab meeting. They are based on collections of model parameters that show unexpected mismatches between `sciunit.score.sort_key` (values that are very close to zero, or approximately zero) and prediction/observation differences that are larger than a factor of ten.